### PR TITLE
Pinned Variable column to the left of the stream table

### PR DESCRIPTION
**Pull request by makaylas**
## Summary/Motivation:
To make the stream table more readable when there is a large number of streams/columns I pinned the Variable column to the left so that when the user scrolls that column stays on the left.

## Changes proposed in this PR:
- Variable column is pinned to the left

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by lbianchi-lbl — APPROVED** [review]
This one-liner LGTM  :+1:

**Comment by ksbeattie** [comment]
@makaylas can you point me to a an example of a flowsheet with a large number of streams to test this with?

**Comment by dangunter** [comment]
@ksbeattie see this gist for an example that creates enough streams to see the behavior. Note that this is from a chemical engineering perspective pure garbage, but can be visualized and has a stream table.

https://gist.github.com/dangunter/f07f9236d2f1c9e0c546e9714199c765

File: idaes/ui/fsvis/static/js/stream_table.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ export class StreamTable {
             let column_header = columns[col]
             if (column_header !== "") {
                 if (column_header === "Variable") {
-                    column_defs.push({headerName: column_header, field: column_header, filter: 'agTextColumnFilter', sortable: true, resizable: true});
+                    column_defs.push({headerName: column_header, field: column_header, filter: 'agTextColumnFilter', sortable: true, resizable: true, pinned: 'left',});
```
Comment (dangunter) [inline idaes/ui/fsvis/static/js/stream_table.js:38]: Why not add a comment saying why this if clause exists?

**Review by dangunter — CHANGES_REQUESTED** [review]
Looks good, but a comment would be nice for future generations.

File: idaes/ui/fsvis/static/js/stream_table.js (hunk 1 above)
Comment (makaylas) [reply idaes/ui/fsvis/static/js/stream_table.js:39]: Added.

**Review by dangunter — APPROVED** [review]
Looks great!

**Review by elbashandy — APPROVED** [review]
Looks good

**Review by ksbeattie — APPROVED** [review]
LGTM

**Pull request merged**